### PR TITLE
Add layout

### PR DESCRIPTION
**Pull request by ArtFlag**
## Description

Add new workman-based layout. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/splitkb/kyria/keymaps/ArtFlag/config.h [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#ifdef RGBLIGHT_ENABLE
+#    define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/splitkb/kyria/keymaps/ArtFlag/config.h:20]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/splitkb/kyria/keymaps/ArtFlag/keymap.c [hunk 2]
```diff
@@ -0,0 +1,43 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/splitkb/kyria/keymaps/ArtFlag/keymap.c:1]: Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/splitkb/kyria/keymaps/ArtFlag/readme.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Simple workman layout
+
+## Features
+
+- Workman layout
+- semicolon and colon are flipped. Colon on tap, semicolon on double tap.
+
+<img src='layout.png'/>
```
Comment (drashna) [inline keyboards/splitkb/kyria/keymaps/ArtFlag/readme.md:8]: Please use markdown rather than html.

ALso, may need to link this to the actual image.

**Review by drashna — CHANGES_REQUESTED** [review]
Also, folders need to be all lower case.

We have a mix of operating systems, and mixed casing causes issues for some systems.

File: keyboards/splitkb/kyria/keymaps/ArtFlag/readme.md (hunk 3 above)
Comment (ArtFlag) [reply keyboards/splitkb/kyria/keymaps/ArtFlag/readme.md]: updated for markdown, but i'm not sure what you mean about linking to the actual image?

Comment (fauxpark) [reply keyboards/splitkb/kyria/keymaps/ArtFlag/readme.md]: Have you looked at the preview?

Comment (ArtFlag) [reply keyboards/splitkb/kyria/keymaps/ArtFlag/readme.md]: nothing wrong the link per se, but I just realized we gitignore png... so I removed the link, thanks for catching this.

File: keyboards/splitkb/kyria/keymaps/artflag/keymap.c [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    if (state->count == 1) {
+        unregister_code16(KC_COLN);
+    } else {
+        unregister_code(KC_SCLN);
+    }
+}
+
+// All tap dance functions would go here.
+qk_tap_dance_action_t tap_dance_actions[] = {
+    [CT_CLN] = ACTION_TAP_DANCE_FN_ADVANCED(NULL, dance_cln_finished, dance_cln_reset),
```
Comment (drashna) [inline keyboards/splitkb/kyria/keymaps/artflag/keymap.c:30]: Just a heads up, you don't need the advanced function here. `ACTION_TAP_DANCE_DOUBLE(KC_SCLN, KC_COLN)` should work here.

Comment (ArtFlag) [reply keyboards/splitkb/kyria/keymaps/artflag/keymap.c:30]: it doesnt seem to work, also this snippet is straight from the docs, so I can send a docs update at the same time if the current solution is not the best one.

Comment (drashna) [reply keyboards/splitkb/kyria/keymaps/artflag/keymap.c:30]: no worries.

**Review by drashna — APPROVED** [review]

**Review by keyboard-magpie — APPROVED** [review]

**Pull request merged**